virtool_workflow/runtime/run_subprocess.py

```diff
@@ -107,12 +106,24 @@ async def _run_subprocess(
 
         async def _stderr_handler(line):
             await stderr_handler(line)
-            logger.info("stderr", line=line.rstrip())
+            line = line.rstrip()
+            try:
+                line_string = line.rstrip().decode()
+                logger.info("stderr", line=line_string)
+            except UnicodeDecodeError as e:
+                logger.exception("decode failed: %s", str(e))
```

Comment:
This will spam the logs when there is a lot of stderr.
```suggestion
            except UnicodeDecodeError:
```